app/src/main/java/ar/com/wolox/android/training/ui/login/LoginFragment.java

```diff
@@ -0,0 +1,117 @@
+package ar.com.wolox.android.training.ui.login;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.View;
+import android.widget.TextView;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.android.training.ui.errors.ErrorCode;
+import ar.com.wolox.android.training.ui.errors.ErrorHandler;
+import ar.com.wolox.android.training.ui.home.HomeActivity;
+import ar.com.wolox.android.training.ui.signup.SignupActivity;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    @BindView(R.id.fragment_login_email) TextView mEmail;
+    @BindView(R.id.fragment_login_password) TextView mPassword;
+
+    @Inject
+    public LoginFragment() {}
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void init() {
+        mEmail.setText(getPresenter().getUserEmail());
+    }
+
+    public void goToHome() {
+        Intent intent = new Intent(getActivity(), HomeActivity.class);
+        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TOP);
+        startActivity(intent);
+        getActivity().finish();
+    }
+
+    @OnClick(R.id.fragment_login_login)
+    public void onLogin() {
+        String email = mEmail.getText().toString();
+        String password = mPassword.getText().toString();
+
+        Boolean validateErrors =    this.validateEmptyFields(email, password)
```

Comment:
Maybe its better to check empty email and if its a valid one , and then check if the password is empty. That allows the user to complete one first.